components/feature/pwa/src/main/java/mozilla/components/feature/pwa/ext/WebAppManifest.kt

```diff
@@ -14,8 +14,21 @@ import androidx.core.net.toUri
 import mozilla.components.browser.state.state.CustomTabConfig
 import mozilla.components.browser.state.state.ExternalAppType
 import mozilla.components.concept.engine.manifest.WebAppManifest
+import mozilla.components.concept.engine.manifest.WebAppManifest.Icon.Purpose
 import mozilla.components.support.utils.ColorUtils.isDark
 
+private const val MIN_INSTALLABLE_ICON_SIZE = 192
```

Comment:
Where did this number come from? Could you document that.

Reply:
"Instability indicators" in the feature brief: https://docs.google.com/document/d/1Dd0OnivaUMe6rvEP5yi6Gt7Hz785ZstDLytaxnXz7Dk/edit

Reply:
FYI: that document is internal so not everyone can see it. A code comment saying, "This is the minimum icon size we want to support" should suffice.

Reply:
I listed out the instability criteria in another comment and I've linked to it from here.